### ISSUE TITLE
Better document and verify the cimfomfa requirement

### ISSUE THREAD

**Issue by ryandesign** [micans/zoem#1]
zoem 21-341 fails to build with many errors like this if cimfomfa is not installed:

```
In file included from ops-xtag.c:9:
In file included from ./ops-xtag.h:12:
./segment.h:12:10: fatal error: 'tingea/ting.h' file not found
#include "tingea/ting.h"
         ^~~~~~~~~~~~~~~
```

After installing cimfomfa 21-361 these problems go away.

Perhaps the zoem configure script could exit with a helpful error message if cimfomfa is not installed?

The ChangeLog says "Compiling zoem now requires installation of cimfomfa" but a person trying to install the software is unlikely to begin by reading the ChangeLog. They are more likely to read the README file, which refers users to the INSTALL file, but neither of these files mention the cimfomfa requirement. Perhaps one or both of those files could mention it?

**Comment by micans** [comment]
Thanks. Is this Debian related (maybe a bit of a random question). For mcl, zoem and aephea should not be needed, depending on how much you want to bootstrap it - I assume you are fully bootstrapping it from autotools sources? If that's not the case it indicates an error in the tarball created, as configure/compile from that tarball should not trigger the documentation build.

**Comment by ryandesign** [comment]
It's MacPorts / macOS related. We have had the latest stable version of [mcl in MacPorts](https://ports.macports.org/port/mcl) for awhile but I noticed it does not build with recent versions of Apple Clang so I'm trying to fix it.

Investigating the latest development version of mcl, I downloaded the latest auto-generated tarball from GitHub. Of course there's no configure script in the repository so I had to bootstrap.
